### ISSUE TITLE
Cold Weather Flying

### ISSUE THREAD

**Issue by the9man** [dalaranwow/dalaran-wow#5830]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Today I've Hit level 77 and went to buy cold weather flying in Borean tundra at Wind Rider Sabamba, located at Warsong Hold in Borean Tundra. For my surprise it is not selling it.

**Expected behaviour**: Tell us what should happen instead
This NPC should have Cold Weather Flying to sell only for players with level 77 and above. (not the tome as dalaran one)
**Steps to reproduce the problem**:

1. Get to level 77.
2. Get to Wind Rider Sabamba, located at Warsong Hold in Borean Tundra.
3. Talk to it.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

Screenshot proof: https://imgur.com/hsDvmMe

The link for the NPC: https://www.wowhead.com/npc=35135/wind-rider-sabamba

The link at fandom (look at Lich King Patch): https://wowwiki.fandom.com/wiki/Cold_Weather_Flying

**Comment by MalkaviaDW** [comment]
Hello there

After Patch 3.2.0, level 80 characters could purchase the heirloom item [Tome of Cold Weather Flight] from Hira Snowdawn in Dalaran. The tome allows alt characters on the same realm and account of level 68 or higher to learn the spell, making flight in Northrend possible from an earlier level.

It is detailed also in the Patch changes:  Patch 3.2.0 (04-Aug-2009): For 1,000Gold level 80 Main characters can purchase the Heirloom Tome of Cold Weather Flight and give it to their alts level 68+ that have a riding skill of 225+ to learn Cold Weather Flying.

We are at 3.3.5a, so it is intended that way. 

Cheers!
